scripts/puppeteer.server.js

```diff
@@ -273,3 +273,23 @@ document.getElementById('members').addEventListener('click', (event) => {
       });
   }
 });
+
+document.getElementById('query-legend').addEventListener('click', (event) => {
+  event.preventDefault();
+  const query = document.getElementById('query').value;
+
+  try {
+    if (event.target.id === 'query-submit') {
+      webexSDKAdapter.peopleAdapter
+        .getPeopleList(query)
+        .subscribe((items) => {
+          console.log('items', items);
+          document.getElementById('people-list').value = JSON.stringify(items, '', 2);
```

Comment:
the second parameter of JSON.stringify is an optional replacer function. The specification says it should be `null` if not needed. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#replacer